### ISSUE TITLE
If you use the 'clear console' button, will it release the memory?

### ISSUE THREAD

**Issue by 0xdevalias** [marianc000/consoleLogMemory#1]
When the console has been opened, and references to that memory are being held onto, if you use the 'clear console' button, will it release the memory?

**Comment by 0xdevalias** [comment]
Just tried this out myself on https://consolelogmemory.onrender.com/

You can see that from 4/5/6 I had the console open, and the memory was increasing. For 7/8 I closed the console, and the memory stopped increasing, but it didn't get released. For 9 I opened the console again, and clicked 'clear console', and you can see the memory got cleaned up, then because I left the console open, it started increasing again:

![image](https://user-images.githubusercontent.com/753891/235806681-9d5c7fd7-a484-4e81-861c-7d567c56b13a.png)

I left the console open, but for 12 I clicked 'clear console' again, which released the memory once more:

![image](https://user-images.githubusercontent.com/753891/235806714-44ff5cdc-dbec-4eb1-8926-8e3779edcb5f.png)

**tl;dr:** Yes, using 'clear console' will release any accumulated memory that was being held onto by the console.
